tests/integration/solidity_contracts/EFTests/test_sha3.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+import pytest
+from starkware.starknet.testing.contract import StarknetContract
+
+from tests.utils.helpers import hex_string_to_bytes_array
+
+logger = logging.getLogger()
+
+
+@pytest.mark.asyncio
+@pytest.mark.EF_TEST
```

Comment:
running locally and i am seeing

```
PytestUnknownMarkWarning: Unknown pytest.mark.EF_TEST - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
    @pytest.mark.EF_TEST

```